fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/Plan.java

```diff
@@ -232,4 +233,8 @@ default String getGroupIdAsString() {
     default String getGroupIdWithPrefix() {
         return "@" + getGroupIdAsString();
     }
+
+    default Plan replaceExpressions(Map<? extends Expression, ? extends Expression> replaceMap) {
```

Comment:
this default impl is dangerous, please impl for all node types. Or Impl it as throw exception